### PR TITLE
Don't cache SQL files when doing "npm run start-dev". Closes #41.

### LINKED ISSUE #33: bot does not actually restart for watch
it shows that it does, but it doesn't!

### LINKED ISSUE #41: Allow auto-reloading of .sql files in "npm run start-dev"
Right now, you need to do `DEBUG=1 npm run start-dev`. This should be simplified.

### PR DESCRIPTION
**Pull request by cowboy**
* Also rename DEBUG env var to DEBUG_SQL and NOCACHE_SQL to address
  @gnarf's comments in #26
* Also closes #33

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -61,7 +61,12 @@ module.exports = function(grunt) {
   });
 
   grunt.registerTask('start', function() {
-    global._BOT = spawn('node', ['--require', 'babel-register', './src/app'], {stdio: 'inherit'});
+    global._BOT = spawn('node', ['--require', 'babel-register', './src/app'], {
+      stdio: 'inherit',
+      env: Object.assign({}, process.env, {
+        NOCACHE_SQL: 1,
```
Comment (tkellen) [inline Gruntfile.js]: i think we should assign this in the actual npm run script

Comment (cowboy) [inline Gruntfile.js]: Ok. any particular reason?

Comment (cowboy) [inline Gruntfile.js]: I'll just do it and not worry about it for now. Explain sometime later if you can.

**Comment by tkellen** [comment]
:+1:

**Pull request merged**